### DISCUSSION TITLE
Instructions for installing postcss-cli do nothing

### DISCUSSION THREAD

**Discussion by philgoetz** [postcss/postcss#1898] [General]
The instructions for installing postcss-cli say only:

`npm i -D postcss postcss-cli`

This produced the response

```
added 65 packages in 11s

14 packages are looking for funding
  run `npm fund` for details
npm notice
npm notice New minor version of npm available! 10.1.0 -> 10.2.3
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.2.3
npm notice Run npm install -g npm@10.2.3 to update!
npm notice
```
However, there is still no postcss executable anywhere that I can find.

**Comment by ai** [comment]
You need to execute PostCSS CLI by calling `npx postcss`

**Reply by philgoetz** [reply]
Thank you!  I would never have figured that out.  The word "npx" doesn't seem to even appear in the docs folder.  How did you figure out how to use npx?  Is there documentation somewhere sufficient for someone to figure out how to use postcss?  'Coz what's here isn't doing it for me.

Also, can I just add node_modules/.bin to my path, instead of running npx (which just opens up a shell with that on my path)?

**Reply by ai** [reply]
It is part of `npm` documentation and using any CLI tool from `node_modules`. Another way is to call `./node_modules/.bin/postcss`.

You can send PR to PostCSS CLI repo (I am maintainer only of the PostCSS core, CLI is on other people).

**Reply by ai** [reply]
> When you say "npm documentation", do you mean https://github.com/postcss/postcss/tree/main/docs ?

No, I mean https://docs.npmjs.com/cli/v7/commands/npx

> https://github.com/postcss/postcss/tree/main/docs 

These docs are about PostCSS as a framework for PostCSS’s plugin authors. It is about parsing CSS and transforming it.

Most end-users use PostCSS in builders like webpack or vite and we left end-user documentation for builder’s authors.
